docs/settingup.md

```diff
@@ -1127,7 +1127,7 @@ Unpublish bookmark previously created in the script with an id `bookmark1`.
 }
 ```
 
-Unpublish bookmark with title "bookmark of testuser", where `testuser` is the username of the simulated user. 
+Unpublish the bookmark with the title "bookmark of testuser", where "testuser" is the username of the simulated user.
```

Comment:
Could you please "highlight" title (`title`), so that it corresponds to the way it is presented under Settings?